### ISSUE TITLE
Supported layers/operators

### ISSUE THREAD

**Issue by amadeuzou** [Tencent/FeatherCNN#2]
Can you provide what full list of layers/operators supported by your engine in each framework (caffe/tensorflow)?

**Comment by HudsonHuang** [comment]
Hi amadeuzou, did you find any list for the operators or layers?

**Comment by Mengjintao** [comment]
Hi amadeuzou, HudsonHuang, 
    Currently, FeatherCNN support 15 caffe layers, they are Conv, depthwise, pointwise, pooling, fully-connection, batchnorm, scale, concat, relu, dropout, slice, eltwise, softmax, prelu, lrn and etc.  Please refer to https://github.com/Tencent/FeatherCNN/blob/master/src/layer_factory.cpp for their definitions. 

   We also provide Filter layer as an example in FeatherCNN to support any user-self-defined layers. Please refer to https://github.com/Tencent/FeatherCNN/wiki/Framework-Extension-Guide-on-User-Defined-Layers for more details. 
  
   What's more,  we plan to provide more official layers for Caffe/Yolo/TensorFlow recently.  I will reply this issue after we finished it.  Thanks for your attention on this work.  

Best,
Jintao Meng
